### DISCUSSION TITLE
Removing duplicate formula?

### DISCUSSION THREAD

**Discussion by rdegges** [Homebrew/discussions#5225] [Everyday usage]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

Hi there! I work at [Snyk](https://snyk.io). Many users use our application via our CLI.

For years now we've maintained a tap for our users to use if they want to install our CLI via brew. You can find our GitHub repo for the tap, as well as all our install instructions here: https://github.com/snyk/homebrew-tap

Here's my question: recently we learned that brew incorporated a Snyk CLI formula directly into core Homebrew. Here's the formula: https://github.com/Homebrew/homebrew-core/blob/master/Formula/s/snyk-cli.rb

What I'm trying to figure out is: should that one be removed? We're officially maintaining our tap as is best practice, and it gets updated frequently. We've had some users wondering which one is the "official" brew package which is what led me here.

Ideally, I think it'd be best to remove the non-Snyk maintained brew formula in favor of our up-to-date tap, but if there's a better solution please let me know! <3

**Comment by SMillerDev** [comment]
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap#official-vendor-taps

**Comment by p-linnane** [comment]
This formula is on our [autobump list](https://github.com/Homebrew/homebrew-core/blob/c5aeee48e2f2539182c8e83537368495dc2531cf/.github/autobump.txt#L1636), meaning our workflow will check for a new release every few hours and open a PR if one exists, so keeping up to date shouldn't be an issue.

You can designate your tap as the "official" one in your own documentation, but the one in homebrew-core will remain available.

**Comment by rdegges** [comment]
Hey @p-linnane, thanks for the info! This is really helpful. Question for you... If the plan is to keep this official formula maintained by the brew team, should we consider deprecating our tap? I'm not sure what the benefit would be to having both of them. If you have any insight I'd appreciate it! <3

**Reply by gromgit** [reply]
> should we consider deprecating our tap?

If you only ever plan to release software the "Homebrew core" way, then there's no point in maintaining your own tap.

But if you plan to support:
* build options
* pre-releases
* multiple release versions, especially violating https://docs.brew.sh/Versions
* [custom dependencies/`Requirement`s](https://docs.brew.sh/Building-Against-Non-Homebrew-Dependencies)

and any other packaging aspects that are [not accepted in Homebrew core](https://docs.brew.sh/Acceptable-Formulae), then you should keep your current tap.

**Reply by p-linnane** [reply]
Agreed with @gromgit. If you are happy with what is being shipped in homebrew-core, we can provide guidance on deprecating your tap and seamlessly migrating your existing tap users to use homebrew-core. Just let us know.
